bluemix-quickstart/cpp/main.cpp

```diff
@@ -43,7 +43,7 @@ extern "C" {
     
     const char * topic = "iot-2/evt/status/fmt/json";
     const char * clientID = strcat(
-            strcpy(new char[50], "d:quickstart:iotquick-edison:"), DEVICE_ID);
+            strcpy(new char[50], "d:quickstart:iotquick-my-iot-board:"), DEVICE_ID);
```

Comment:
This sample still has the old reference board check. Is that for next week?

Reply:
yes